pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -274,10 +284,11 @@ func (c *Controller) Run(ctx context.Context, workers int) {
 		return
 	}
 
-	logger.V(2).Info("Starting worker threads", "total", workers)
+	logger.V(2).Info("Starting worker threads", "total", workers + 1)
```

Comment:
Nit: I'd actually leave this number as it is and add analogous log just before you start the topology cache-related goroutine.

Reply:
Done